schema.graphql

```diff
@@ -7,6 +7,8 @@ type Collection @entity {
   freezer: String
   metadataCid: String
   isDestroyed: Boolean!
+  createdAt: Float @index
+  updatedAt: Float @index
```

Comment:
Can `Float` data be a good indexer?

Reply:
I though this is okay for us, it's more simple than a Scalar Date, the Float type is because sometimes the `new Date().getTime()` had decimal places, so to prevent warnings or the indexer gets stoped